common/src/main/java/org/apache/rocketmq/common/protocol/header/GetEarliestMsgStoreTimeResponseHeader.java

```diff
@@ -16,15 +16,15 @@
  */
 
 /**
- * $Id: GetEarliestMsgStoretimeResponseHeader.java 1835 2013-05-16 02:00:50Z vintagewang@apache.org $
+ * $Id: GetEarliestMsgStoreTimeResponseHeader.java 1835 2013-05-16 02:00:50Z vintagewang@apache.org $
```

Comment:
There are many files with such lines. If you don't mind, let's have them removed with this commit ;)